Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py

```diff
@@ -0,0 +1,556 @@
+            return:
+             Dict: represents human readable markdown table
+             int: dbot score
+    """
+
+    threshold_score = SEVERITY_SCORE.get(threshold)
+    if not threshold_score:
+        raise Exception(
+            f'Cofense error: Invalid threshold value: {threshold}. Valid values are: None, Minor, Moderate or Major')
+
+    md_data = []
+    indicator_found = False
+    dbot_score = adjusted_score = 0
+
+    for threat in threats:
+        severity_level = 0
+        for block in threat.get('blockSet'):
+
+            if block.get('impact'):
+                threat_score: int = SEVERITY_SCORE.get(block.get('impact'), 0)
+                adjusted_score = 3 if threshold_score <= threat_score else threat_score
+                if block.get('data') == indicator:
+                    dbot_score = severity_level = adjusted_score
+                    indicator_found = True
+                    break
+
+            severity_level = max(severity_level, adjusted_score)
+
+        md_data.append(create_threat_md_row(threat, severity_level))
+
+        if not indicator_found:
+            dbot_score = max(dbot_score, severity_level)
+
+    return md_data, dbot_score
+
+
+def ip_threats_analysis(threats: List, ip: str, threshold: str, dbot_score_obj):
+    """ process raw response data and generate dbot score ,human readable results, ip indicator object
+            Args:
+                - threats (list): threats data from cofense raw response
+                - indicator (string): threat severity level for dbot score calculation
+                - threshold (string): threshold for threat's severity
+            return:
+             Dict: represents human readable markdown table
+             int: dbot score
+             ip indicator : indicator object with the data collected from the threats
+    """
+    threshold_score = SEVERITY_SCORE.get(threshold)
+    if not threshold_score:
+        raise Exception(
+            f'Cofense error: Invalid threshold value: {threshold}. Valid values are: None, Minor, Moderate or Major')
+
+    md_data = []
+    indicator_found = False
+    dbot_score = adjusted_score = severity_level = 0
+    ip_indicator = Common.IP(ip=ip, dbot_score=dbot_score_obj)
+    for threat in threats:
+        severity_level = 0
+        for block in threat.get('blockSet'):
+            if block.get('data') == ip and block.get('ipDetail'):
+                ip_indicator.asn = block.get('ipDetail').get('asn')
+                ip_indicator.geo_latitude = block.get("ipDetail").get("latitude")
+                ip_indicator.geo_longitude = block.get("ipDetail").get("longitude")
+                ip_indicator.geo_country = block.get("ipDetail").get("countryIsoCode")
+
+            if block.get('impact'):
+                threat_score = SEVERITY_SCORE.get(block.get('impact'), 0)
+                adjusted_score = 3 if threshold_score <= threat_score else threat_score
```

Comment:
the `break` found in line 191 can cause some inconsistencies in the way the score is calculated.
it might be more correct to do 2 for loops one for score and one for additional IP details the same way you are doing for files.